boards/qemu_i486_q35/Makefile

```diff
@@ -0,0 +1,48 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright Tock Contributors 2024.
+
+# Makefile for building the Tock kernel for the qemu-system-i386 `q35`
+# platform / machine type.
+
+# Skip auto-installing targets with rustup, since we are using a custom target
+NO_RUSTUP := 1
+
+include ../Makefile.common
+
+QEMU_CMD             := qemu-system-i386
+WORKING_QEMU_VERSION := 7.2.0
+
+# Peripherals attached by default:
+# - 16550 UART (attached to stdio by default)
+QEMU_BASE_CMDLINE := \
+  $(QEMU_CMD) \
+    -cpu 486 \
+    -machine q35 \
+    -net none \
+    -nographic
+
+# Run the kernel inside a qemu-system-i386 "q35" machine type simulation
+#
+# In order to boot with QEMU's -kernel flag, the Multiboot V1 header (at the beginning of .text)
+# must appear within the first 8k of the ELF file. Unfortunately, the ELF emitted by Rust/LLVM does
+# not always place .text first.
+#
+# To address this issue, we use GNU objcopy with no arguments to regenerate the ELF file. This does
+# not change the binary properties in any meaningful way, but it does reorder the physical offset of
+# sections within the ELF file such that .text comes first and the Multiboot header is discoverable.
+.PHONY: run
+run: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
+	@echo
+	@echo Reordering ELF file sections
+	@/opt/homebrew/opt/binutils/bin/objcopy $<
```

Comment:
Oops! No absolute Mac-specific path please!

I think assuming `objcopy` is on-path. Just `objcopy` works for me (it's in my path), but perhaps is sensitive to the host system being x86?

Reply:
> Oops! No absolute Mac-specific path please!
> 
> I think assuming `objcopy` is on-path. Just `objcopy` works for me (it's in my path), but perhaps is sensitive to the host system being x86?

Perhaps if you are worried about the default target being used,
you could specify the target explicitly (I would assume) `objcopy --target elf32-i386`?

Reply:
I don't have an `objcopy` executable in my path on mac, just fyi.

Reply:
FWIW, I did `brew install binutils` and then could modify the path to `/usr/local/opt/binutils/bin/objcopy`.